plugin/core/sessions.py

```diff
@@ -1469,9 +1469,18 @@ def can_handle(self, view: sublime.View, scheme: str, capability: str | None, in
                 return self.has_capability(capability)
         return False
 
-    def has_capability(self, capability: str) -> bool:
+    def has_capability(self, capability: str, *, include_local: bool = False) -> bool:
```

Comment:
I'm just thinking whether "local" wording is clear enough... Would it be better to say `include_views` or `include_buffers`?